standalone-dns-proxy/cmd/root.go

```diff
@@ -46,9 +45,7 @@ func NewDNSProxyCmd(h *hive.Hive) *cobra.Command {
 		Use:   binaryName,
 		Short: "Run " + binaryName,
 		Run: func(cobraCmd *cobra.Command, args []string) {
-			logger := logging.DefaultSlogLogger.With(logfields.LogSubsys, binaryName)
-
-			initEnv(logger, h.Viper())
+			initEnv(logging.DefaultSlogLogger, h.Viper())
```

Comment:
nit: This hunk (and others in this file) were squashed into the second commit, but they belong in the first commit.